runtime/drivers/pinot/sqldriver/driver.go

```diff
@@ -10,13 +10,65 @@ import (
 	"io"
 	"math"
 	"math/big"
+	"net"
 	"net/url"
 	"reflect"
+	"regexp"
+	"strconv"
 	"time"
 
+	"github.com/rilldata/rill/runtime/pkg/retrier"
 	"github.com/startreedata/pinot-client-go/pinot"
 )
 
+func isRetryableHTTPError(err error) bool {
```

Comment:
Move these util functions to the end of the file as per Uber's Go style guide, which we follow: https://github.com/uber-go/guide/blob/master/style.md#function-grouping-and-ordering

I know in some languages it's common to have utils before the main logic, but we (and most other Go projects) order files with the external/main/important logic first.